automation/orchestrator.go

```diff
@@ -68,8 +68,7 @@ type Orchestrator struct {
 	running      bool
 }
 
-// NewOrchestrator constructs an orchestrator, whose only responsibility, right
-// now, is to create a workflow store, run store, & listen for trigger events
+// NewOrchestrator constructs an orchestrator
```

Comment:
this is weird commentary from when i was iteratively adding bits to the orchestrator